mrp_multi_level/wizards/mrp_inventory_procure.py

```diff
@@ -83,8 +83,8 @@ def make_procurement(self):
                     item.qty,
                     item.uom_id,
                     item.location_id,
-                    "MRP: " + item.planned_order_id.name or str(self.env.user.login),
-                    "MRP: " + item.planned_order_id.origin or str(self.env.user.login),
+                    "MRP: " + str(item.planned_order_id.name) or str(self.env.user.login),
+                    "MRP: " + str(item.planned_order_id.origin) or str(self.env.user.login),
```

Comment:
```suggestion
                    "MRP: " + (item.planned_order_id.name or "") str(self.env.user.login),
                    "MRP: " + (item.planned_order_id.origin or "") or str(self.env.user.login),
```
str(None) = 'None'
str(False) = 'False'

I suppose you don't want a string like "MRP: False" :)

Reply:
Also you need to execute `pre-commit run -a` in your command window